tests/unit/components/saphyr/SaphyrEnzymeModal.spec.js

```diff
@@ -39,7 +39,7 @@ describe('SaphyrEnzymeModal.vue', () => {
     it('has enzyme options', async () => {
       let enzymeOptions = {
         enzymeOptions: [
-          { value: null, text: 'Please select an option' },
+          { value: '', text: 'Please select an option' },
```

Comment:
This variable is a bit weird. Maybe remove the enzymeOptions object and keep the array. Then change lines 48/49 to be `wrapper.setData({enzymeOptions: enzymeOptions})
      expect(enzymeModal.enzymeOptions).toEqual(enzymeOptions)`